main.py

```diff
@@ -97,12 +112,68 @@ def _remove_submission_exceeding_threshold(self, submission, service):
                                                                     service.listeners_count)
         submission.mod.remove(False, mod_message)
         self._reply_with_sticky_post(submission, reply)
+    
+    def _perform_repost_mod_action(self, submission, template):
+        if self.config.REMOVE_SUBMISSIONS is True:
+            self._remove_repost(submission, template)
+        else:
+            submission.report("repost :(")
+    
+    def _remove_repost(self, submission, template):
+        reply = template
+        submission.mod.remove()
+        self._reply_with_sticky_post(submission, reply)
+
+    def _process_successful_post(self):
+        new_id_list = RepostChecker.get_submissions_before_time(time.time() - 252000)
+        for post_id in new_id_list:
+            old_submission = self.reddit.submission(id=post_id[0])
+            old_score = old_submission.score
+            if old_score > 100:
+                artist = TitleParser.create_parsed_submission_from_submission(old_submission)
+                RepostChecker.add_count(TitleParser.get_prioritized_artist(artist).lower())
```

Comment:
I see you're calling the repost checker staticly here and throughout this pr? :thinking: